### PR TITLE
Minor usability fixes

### PR DESCRIPTION
**Pull request by bver**
* `--task` option can specify arbitrary `./data` subdir (not only `atis` or `snips`)
* transformers.models use updated

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -103,5 +104,22 @@ def get_sentence_frame_acc(intent_preds, intent_labels, slot_preds, slot_labels)
 
     sementic_acc = np.multiply(intent_result, slot_result).mean()
     return {
-        "sementic_frame_acc": sementic_acc
+        "semantic_frame_acc": sementic_acc
```
Comment (jarbes) [inline utils.py:107]: Thank you. It was painful to watch this typo :)

File: utils.py [hunk 2]
```diff
@@ -103,5 +104,22 @@ def get_sentence_frame_acc(intent_preds, intent_labels, slot_preds, slot_labels)
...
+    def __init__(self, csv_filename):
+        self.csv_file = open(csv_filename, 'w')
+        self.writer = None
+
+    def write_results(self, results):
+        if not self.writer:
+            self.writer = csv.DictWriter(self.csv_file, fieldnames=results.keys())
+            self.writer.writeheader()
+        self.writer.writerow(results)
+        self.csv_file.flush()
```
Comment (jarbes) [inline utils.py:122]: So even if I stopped the training by killing it prematurely, the training stats would be saved in the csv file. Is that what `flush` does?

**Review by jarbes — APPROVED** [review]
Looks good. Thank you!

File: utils.py (hunk 2 above)
Comment (bver) [reply utils.py:122]: Correct. The file is properly closed on the end of training but flush guarantees we do not lose the metrics data if the process is terminated prematurely.

**Pull request merged**